### ISSUE TITLE
Support codedeploy import app

### ISSUE THREAD

**Issue by agutierrezl** [terraform-providers/terraform-provider-aws#1473]
Hi there,

This is a feature request. It will be nice you included support for importing code deploy apps.

### Terraform Version
$ terraform -v
Terraform v0.10.2

### Affected Resource(s)
Please list the resources as a list, for example:
- aws_codedeploy_app
- aws_codedeploy_deployment_group

### Terraform Configuration Files
```hcl
resource "aws_codedeploy_app" "app" {
  name = "${var.project}"
}

resource "aws_codedeploy_deployment_group" "deployment-group" {
  app_name = "${aws_codedeploy_app.app.name}"
  deployment_group_name = "${var.project}-App"
  service_role_arn = "${aws_iam_role.codedeploy_iam_role.arn}"
  auto_rollback_configuration {
    enabled = false
  }
  autoscaling_groups = ["${aws_autoscaling_group.asg.name}"]
  deployment_config_name = "CodeDeployDefault.OneAtATime"
}
```

### Expected Behavior
CodeDeploy app imported to the state file

### Actual Behavior
aws_codedeploy_app.app: Importing from ID "aws-codedeploy-app"...
Error importing: 1 error(s) occurred:

* aws_codedeploy_app.app (import id: aws-codedeploy-app): import aws_codedeploy_app.app (id: aws-codedeploy-app): resource aws_codedeploy_app doesn't support import

...
aws_codedeploy_deployment_group.deployment-group: Importing from ID "dg-App"...
Error importing: 1 error(s) occurred:

* aws_codedeploy_deployment_group.deployment-group (import id: dg-App): import aws_codedeploy_deployment_group.deployment-group (id: dg-App): resource aws_codedeploy_deployment_group doesn't support import

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform import aws_codedeploy_app.app aws-codedeploy-app`
2. `terraform import aws_codedeploy_deployment_group.deployment-group dg-App`

**Comment by hegyre** [comment]
Hello guys,

I'm in the same situation, no way to import a deployment group

**Comment by bflad** [comment]
Pull request submitted: #6025

**Comment by bflad** [comment]
Support for `aws_codedeploy_*` resource import has been merged and will release with version 1.39.0 of the AWS provider, likely later today.
